### PR TITLE
added username support

### PR DESCRIPTION
**Pull request by IlayMenachem**
more to come, this is just adding the username to the template.

### CODE REVIEW

File: jupyterlab_templates/extension.py [hunk 1]
```diff
@@ -33,7 +33,9 @@ def get_templates(self):
             for f, dirname, filename in files:
                 with open(os.path.join(abspath, f), 'r', encoding='utf8') as fp:
                     content = fp.read()
-                templates[os.path.join(dirname, filename)] = {'path': f, 'dirname': dirname, 'filename': filename, 'content': content}
+                templates[os.path.join(dirname, filename)] = {'path': f, 'dirname': dirname, 'filename': filename,
+                                                              'content': content.replace("##username##", username),
```
Comment (ofirassif) [inline jupyterlab_templates/extension.py]: Const ###username### out

File: jupyterlab_templates/extension.py [hunk 2]
```diff
@@ -55,10 +57,17 @@ def initialize(self, loader):
         self.loader = loader
 
     def get(self):
-        template_names = self.loader.get_templates().keys()
+        template_names = self.loader.get_templates(username(self)).keys()
         self.finish(json.dumps(sorted(template_names)))
 
 
+def username(web_handler):
```
Comment (ofirassif) [inline jupyterlab_templates/extension.py]: get_username?

File: jupyterlab_templates/extension.py [hunk 3]
```diff
@@ -55,10 +57,17 @@ def initialize(self, loader):
...
+    data = web_handler.get_current_user()
+    if data == 'anonymous':
```
Comment (ofirassif) [inline jupyterlab_templates/extension.py]: Const out the default username

**Review by ofirassif — CHANGES_REQUESTED** [review]

File: jupyterlab_templates/extension.py (hunk 1 above)
Comment (IlayMenachem) [reply jupyterlab_templates/extension.py]: fixed

File: jupyterlab_templates/extension.py (hunk 2 above)
Comment (IlayMenachem) [reply jupyterlab_templates/extension.py]: fixed

File: jupyterlab_templates/extension.py (hunk 3 above)
Comment (IlayMenachem) [reply jupyterlab_templates/extension.py]: fixed

**Pull request merged**